pallets/ocw-demo/src/lib.rs

```diff
@@ -196,6 +217,11 @@ decl_module! {
 			debug::info!("submit_number_unsigned: {}", number);
 			Self::append_or_replace_number(number);
 
+			// Off-chain indexing write
+			let key = Self::derived_key(frame_system::Module::<T>::block_number());
+			let data = IndexingData(b"submit_number_unsigned".to_vec(), number);
+			offchain_index::set(&key, &data.encode());
```

Comment:
@tomusdrw I am forming the key hash based on the pallet predefined prefix and the block number, and use it to store some data now.

Reply:
Cool, looks good. Like it's still possible that multiple forks can override this data, but there isn't a very good solution in this particular example. And also given that we are going to write exactly the same data in forks (it only relies on the `number`) it's good enough :)